.github/workflows/editorconfig.yml

```diff
@@ -10,6 +10,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
```

Comment:
Seems to be a bit quicker to do this first rather than letting the `get-diff-action` do it.

Reply:
Might be a good idea to see about rewriting git history to remove checked in tarballs, binaries, etc.

Reply:
Has anybody committed this in the past?

Reply:
I looked a while back out of curiosity, IIRC there was at least one large tarball and a few other things that probably shouldn't have been checked in.

I think I saved the results somewhere, I'll see if i can find them or I'll recheck the repo.

Reply:
I can't find the original but quick scan (excluding nix & json files) shows:

```
77ff01cf186e  1.0MiB pkgs/stdenv/linux/tools/bash
ff689f7e31a9  1.7MiB pkgs/stdenv/nix-linux-static/bash-static/bash-2.05b-static.tar.gz
f57b203eee2c  2.6MiB pkgs/stdenv/nix-linux-static/gcc-static/gcc-3.3.4-static.tar.gz
c7207aec8964  3.5MiB pkgs/os-specific/linux/blcr/fixes.patch
dc1d9b1f1784   39MiB pkgs/stdenv/linux/bootstrap-tools.cpio.bz2
```

Reply:
Rewriting history sound quite bad though. Is it even possible do this in non-destructive way?

Reply:
No, I don't think it's possible.